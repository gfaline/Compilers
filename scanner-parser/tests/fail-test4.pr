fn foo() -> void
{
  string foo_;
  foo_ = false;
 
}