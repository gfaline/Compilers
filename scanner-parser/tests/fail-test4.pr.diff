diff: fail-test4.pr.err: No such file or directory
diff: testss/tests/fail-test4.pr.err: No such file or directory
