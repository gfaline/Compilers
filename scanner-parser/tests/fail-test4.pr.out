tests/fail-test4.pr
